api/client/transform/fortio.proto

```diff
@@ -34,6 +34,8 @@ message FortioResult {
   map<string, uint64> RetCodes = 11;
 
   string URL = 12;
+
+  string Version = 13;
```

Comment:
You could make this a structured type, @yanavlasov is introducing this in https://github.com/envoyproxy/envoy/pull/9301

Reply:
+1